.gitignore

```diff
@@ -0,0 +1,4 @@
+# directories
```

Comment:
could you remove this file from the commit? Is relevant for you locally because you have this folder but not for other people using this repository

Reply:
yes shouldn't be here, my bad